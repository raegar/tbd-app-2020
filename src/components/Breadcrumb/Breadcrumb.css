ul.breadcrumb {
    padding: 10px 16px;
    list-style: none;
    background-color: #31B7BC;  justify-content: center; align-items: center; display: flex;}

ul.breadcrumb li {
  display: inline;
  font-size: 18px;
}

ul.breadcrumb li+li:before {
  padding: 8px;
  color: white;
  content: "/\00a0";
  
}

ul.breadcrumb li a {
  color:white;
  text-decoration: none;
}

ul.breadcrumb li a:hover {
  color: white;
  text-decoration: underline;
}
