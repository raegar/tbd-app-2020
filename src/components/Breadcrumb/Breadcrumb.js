import React from 'react';
import { Container } from 'react-bootstrap';
import './Breadcrumb.css';
//import { Breadcrumb } from 'react-bootstrap';

class BreadcrumbComp extends React.Component{
	render = () => {
		return (
			<Container>
				<ul class="breadcrumb">
					<li><a href="www.google.co.uk">Personal Information</a></li>
					<li><a href="www.google.co.uk">Qualifications</a></li>
					<li><a href="www.google.co.uk">About you</a></li>
				</ul>
			</Container>
		);
	}
};

export default BreadcrumbComp;