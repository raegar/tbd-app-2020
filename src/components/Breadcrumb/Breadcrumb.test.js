import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import Breadcrumb from './Breadcrumb';
import { BrowserRouter as Router} from 'react-router-dom';

//cleans up 
afterEach(cleanup);

it("Renders Breadcrumb", () => {
    const { asFragment } = render(<Router><Breadcrumb /></Router>);
    expect(asFragment()).toMatchSnapshot();
});