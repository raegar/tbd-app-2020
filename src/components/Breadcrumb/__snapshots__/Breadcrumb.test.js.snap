// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Breadcrumb 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <ul
      class="breadcrumb"
    >
      <li>
        <a
          href="www.google.co.uk"
        >
          Personal Information
        </a>
      </li>
      <li>
        <a
          href="www.google.co.uk"
        >
          Qualifications
        </a>
      </li>
      <li>
        <a
          href="www.google.co.uk"
        >
          About you
        </a>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
